src/client/Microsoft.Identity.Client/AppConfig/ConfidentialClientApplicationBuilder.cs

```diff
@@ -203,11 +204,6 @@ public ConfidentialClientApplicationBuilder WithClientAssertion(Func<string> cli
         /// <returns>The builder to chain the .With methods</returns>
         public ConfidentialClientApplicationBuilder WithAzureRegion(string azureRegion = ConfidentialClientApplication.AttemptRegionDiscovery)
```

Comment:
>azureRegion [](http://example.com/codeflow?start=75&length=11)

I would have assumed we should fail if null is used. This way people can believe they are using regional but they are not.

Reply:
I'm pretty sure this fails with an `ArgumentNullException`

Reply:
So, @henrik-me, shall I close this PR and the associated bug ?

Reply:
but i don't see a test checking for this? unless i missed it.